### PR TITLE
Use apt file for installing dependencies

### PR DESCRIPTION
**Pull request by pattacini**
The only doubt I have is regarded w/ `liblua5.1-dev`, which I didn't find in the `apt.txt`.
Not sure if it's needed. As of now, I've removed it.

### CODE REVIEW

**Comment by pattacini** [comment]
Test is underway in https://github.com/icub-training/icub-training.github.io/actions.

File: dockerfiles/Dockerfile [hunk 1]
```diff
@@ -68,9 +55,22 @@ ARG ROBOTOLOGY_SUPERBUILD_INSTALL_DIR=/usr/local
 RUN git config --global user.name "GitHub Actions" && \
     git config --global user.email "actions@github.com"
 
-# Build robotology-superbuild
+# Install dependencies
 RUN git clone https://github.com/robotology/robotology-superbuild.git --depth 1 --branch ${ROBOTOLOGY_SUPERBUILD_RELEASE} && \
-    cd robotology-superbuild && \
+    sudo robotology-superbuild/scripts/install_apt_dependencies.sh
```
Comment (pattacini) [inline dockerfiles/Dockerfile:60]: I'm also trying out this command without explicitly resorting to `bash`.

**Comment by pattacini** [comment]
Image built successfully ✔️

**Comment by traversaro** [comment]
> The only doubt I have is regarded w/ `liblua5.1-dev`, which I didn't find in the `apt.txt`.

That (or any equivalent for different versions of lua) is only necessary if you enable `ROBOTOLOGY_USES_LUA` .

**Comment by pattacini** [comment]
Actually, the package I missed was `lua5.1`.
See https://github.com/robotology/robotology-superbuild/pull/876 too.

**Pull request merged**